src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -20,6 +21,8 @@ public TypedParameter(string parameter, ProtoCore.Type type, AssociativeNode def
             Name = parameter;
             Type = type;
             DefaultValue = defaultValue;
+
+            defaultValueString = shortArgumentName;
```

Comment:
`defaultValueString` is the same as `defaultValue.ToString()` (in LibraryServices.cs, line 822), so can we just use `defaultValue.ToString()`?